src/domain/reviews/cancellations/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { RouteComponentProps } from "@reach/router"
+import { navigate } from "gatsby"
+import { parse } from "query-string"
+import React from "react"
+import BodyCard from "../../../components/organisms/body-card"
+import CancellationFeedbackTable from "../../../components/templates/cancellation-feedback-table"
+import { useCancellationFeedbackQuery } from "../../../services/nibll-api"
+import ReviewsHeader from "../header"
+
+const CancellationFeedbacks: React.FC<RouteComponentProps> = ({ location }) => {
+  const queryParams = parse(location?.search as string)
```

Comment:
Is parse already part of the project? It's pretty easy to do `const queryParams = new URLSearchParams(location?.search || '');` and then do `const keyValue = queryParams.get(keyName)`.

Reply:
Yea `parse` is already part of the project. I didn't add it